code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -105,9 +107,11 @@
 	push_data()
 	activate_pin(2)
 
+
+
 /obj/item/integrated_circuit/input/adv_med_scanner
 	name = "integrated advanced medical analyser"
-	desc = "A very small version of the common medical analyser.  This allows the machine to know how healthy someone is.  \
+	desc = "A very small version of the medbot's medical analyser.  This allows the machine to know how healthy someone is.  \
```

Comment:
Why was this changed to specify medbot?

Reply:
but medbot can remotely scan your health.
I don't know, where is logic, but there is a fact: if you want to recreate medbot's functions with circuitry, you'll need remote health scan.